samples/WebFormsToBlazor/BlazorCore/Program.cs

```diff
@@ -1,27 +1,62 @@
 using BlazorCore;
-using BlazorCore.Data;
+using BlazorCore.Components;
 using Microsoft.AspNetCore.Components.Web;
 
 var builder = WebApplication.CreateBuilder(args);
 
-builder.Services.AddSystemWebAdapters();
-builder.Services.AddReverseProxy();
+// Add YARP
+builder.Services.AddHttpForwarder();
+
+// Add constraint to redirect .axd files to WebForms
+builder.Services.AddRouting(options => options.ConstraintMap.Add("isAxdFile", typeof(AxdConstraint)));
+// Add System Web Adapters and setup session
+builder.Services.AddSystemWebAdapters()
+    .AddJsonSessionSerializer(options =>
+    {
+        options.RegisterKey<string>("test-value");
+    })
+    .AddRemoteAppClient(options =>
+    {
+        options.RemoteAppUrl = new(builder.Configuration["ProxyTo"]);
+        options.ApiKey = builder.Configuration["RemoteAppApiKey"];
+    })
+    .AddSessionClient();
 
 // Add services to the container.
-builder.Services.AddRazorPages();
-builder.Services.AddServerSideBlazor(options =>
-{
-    options.RootComponents.RegisterCustomElement<HelloWorld>("hello-world");
-});
-builder.Services.AddSingleton<WeatherForecastService>();
+builder.Services.AddRazorComponents()
+    .AddInteractiveServerComponents(options => options.RootComponents.RegisterCustomElement<HelloWorld>("hello-world"));
 
 var app = builder.Build();
 
+app.UseHttpsRedirection();
+
 app.UseStaticFiles();
-app.UseRouting();
+app.UseAntiforgery();
+
+/*
+ * When Blazor interactive SSR establishes a SignalR connection, a CONNECT method is requested.
+ * This request will stay open until Blazor sends a disconnect request. 
+ * Since it stays open, the session is not properly committed and will stay in the heartbeat loop on the framework side of System.WebAdapters.
+ * We can either:
+ *     A: Block all connect requests from using System.Web.Adapters
+ *     B: Block all requests from paths containing "/_blazor"
+ * TLDR: We need to ensure that SystemWeb.Adapters is not used with Blazor SignalR
+ */
+app.UseWhen(
+    (context) => HttpMethods.IsConnect(context.Request.Method) == false,
+    appBuilder => appBuilder.UseSystemWebAdapters());
+//app.UseWhen(
+//    (context) => context.Request.Path.ToString().Contains("/_blazor", StringComparison.OrdinalIgnoreCase) == false,
+//    appBuilder => appBuilder.UseSystemWebAdapters());
 
-app.MapBlazorHub();
-app.MapBlazorPages("/_Host");
-app.MapForwarder("/{**catch-all}", app.Configuration["ProxyTo"]!).WithOrder(int.MaxValue);
+app.MapRazorComponents<App>()
```

Comment:
This breaks the existing sample that is to show using blazor components in webforms. It seems like this sample you have should be a separate one

Reply:
The existing sample still works. See the images below. The blazor web page is now the index and I have updated the nav bar to reference web form pages. 

![image](https://github.com/user-attachments/assets/8af1a6c8-55a5-4f7c-86a0-4e590688e689)
 
![image](https://github.com/user-attachments/assets/2927110b-8b21-43df-8fea-46d2ae4da0bb)

I can make a separate sample if needed. I was under the impression from my previous PR that I should combine them.

Reply:
awesome! I did not see that initially. it's fine as-is